src/bits.c

```diff
@@ -198,4 +201,20 @@ void bitcopy(void *dst, u32 dst_index, const void *src, u32 src_index,
   }
 }
 
+/**
+ * Count number of bits set in 32 bit value
+ *
+ * \param[in]     v   32 bits value
+ * \param[in]     bv  1 or 0 - which value to count
+ *
+ * \return        Number of bits set to one or zero.
+ */
+u8 get_bits_number(u32 v, u8 bv)
```

Comment:
Can we rename this to `count_bits_u32()` or something a little more descriptive?